Input flags enhancement (#109)
ZupIT/docs-ritchie: content/en/Formulas/Rename formulas.md

{{% alert color="info" %}}

- Ritchie will automatically identify the workspace to which the formula belongs.

>   - If you identify more than one workspace with the entered formula, an extra necessary step (_when using prompt execution_) will be to choose in which workspace the formula should have its name changed.

Reviewer: ```suggestion
  - If you identify more than one workspace with the entered formula, when using prompt execution, Ritchie will choose which workspace the formula should have its name changed.
```

Author: Vou ter que fazer o ajuste aqui, que de fato parece que é o usuário que identifica o workspace. Não prestei atenção ao alterar a definição.
